packages/api/src/qm/visitor.ts

```diff
@@ -89,7 +89,7 @@ const questionVisitor = async function (
   totalSteps?: number
 ): Promise<Result<InputResult<any>, FxError>> {
   if (inputs[question.name] !== undefined) {
-    return ok({ type: "success", result: inputs[question.name] });
+    return ok({ type: "skip", result: inputs[question.name] });
```

Comment:
@jayzhang I was considering removing the create question from question model but has side effect since the download sample logic will also traverse the question model. So I have to keep the question model for create and make the question as 'skip' if the answer to the question is pre-set. Do you have any concern?